ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -385,3 +390,112 @@ def set_upgrade_images(self):
             with NamedTemporaryFile() as cs_yaml:
                 dump_data_to_temp_yaml(cs_data, cs_yaml.name)
                 ocs_catalog.apply(cs_yaml.name)
+
+
+class UpgradeDisruptiveOps(EcosystemTest):
+    """
+    Contains disruptive upgrade tests functions
+
+    """
+
+    def get_ocs_operator_node_name(self):
+        """
+        Getting node's name that running ocs-operator pod
+
+        Returns:
+            str: node's name that running ocs-operator pod
+
+        """
+        ocs_operator_pod = get_ocs_operator_pod()
+        log.debug(f"ocs operator pod info: {ocs_operator_pod}")
+        ocs_operator_node = get_pod_node(ocs_operator_pod)
+
+        return get_node_name(ocs_operator_node)
+
+    def worker_node_shutdown(self, nodes, abrupt):
```

Comment:
Is this function only upgrade specific? Or can this live directly in the node related module  `ocs_ci.ocs.node` ? Maybe also the function above is not really upgrade specific and can live directly in node module and used from there?

Reply:
There is more functions like this planned, and we can use it for other things then upgrade. maybe new module for disruptive function would be better?

Reply:
Sounds good